compiler/main/driver.cpp

```diff
@@ -665,6 +572,11 @@ static void setPrintPassesFile(const ArgumentState* state, const char* fileName)
   }
 }
 
+static void setLocal (const ArgumentDescription* desc, const char* unused) {
+  // Used in postLocal
+  fSetLocal = true;
```

Comment:
`fSetLocal` was a little confusing to me at first. Might be a little more obvious as `fUserSetLocal` and/or changing "Used in postlocal" to something like "need to know if user threw --[no-]local after arg processing to set `flocal`"